concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -313,9 +313,25 @@ public Semver getVersion() {
         return new Semver(versionString);
     }
 
+    /**
+     * Get the list of ips that the node will currently not connect to, not accept connections for.
```

Comment:
It's more than just the IPs, it's in fact a list of `Peer`s. 
```suggestion
     * Get a list of banned peers.
```